docs/data-cookbooks/cnm-workflow.md

```diff
@@ -469,15 +473,15 @@ The data written to the `CnmResponseStream` should adhere to the [Response Messa
   "identifier": "1234-abcd-efg0-9876",
   "response": {
     "status": "FAILURE",
-    "errorCode": "INGEST_ERROR",
+    "errorCode": "PROCESSING_ERROR",
```

Comment:
I'm debating whether we should remove the example CNM input records and responses? They duplicate what is on the CNM wiki page, so we're just increasing the chance of documentation drift. Maybe best to just point at the wiki page for the examples input / output for CNM?

Reply:
I'm not sure the wiki pages are accessible for everyone. Then again, if they're not accessible to you then you're probably not using CNM. In that case, then probably best to point at the wiki.

Reply:
As discussed, let's leave these in for now pending further work on CNM in 20.1